content/document.js

```diff
@@ -15,7 +15,8 @@ const {
 const { getPopularities } = require("./popularities");
 const { getWikiHistories } = require("./wikihistories");
 
-const { memoize, slugToFolder } = require("./utils");
+const { buildURL, memoize, slugToFolder, execGit } = require("./utils");
+const Redirects = require("./redirect");
```

Comment:
It's weird that you're importing a plural from a singular. 
If `Redirects` is the right namespace name, perhaps the file should be renamed to `content/redirects.js` to make it less "weird".